### ISSUE TITLE
setColor followed by resize breaks the picker

### ISSUE THREAD

**Issue by jimparis** [jaames/iro.js#156]
If you visit the demo at https://codepen.io/rakujira/pen/WZOeNq
and add the following to the bottom of the JS:
```js
colorPicker.setColors(["#0000ff", '#00ff00']);
colorPicker.resize(200);
```
then the picker breaks (try dragging the handle).
Either one of those lines alone works as expected.  Or wrapping the second line in a setTimeout like:
```js
colorPicker.setColors(["#0000ff", '#00ff00']);
setTimeout(() => { colorPicker.resize(200); }, 0);
```

**Comment by jimparis** [comment]
May be related to #153

**Comment by jaames** [comment]
Thanks for the report! Sorry for taking so long to get to this, it should be fixed in the next release (v 5.5.0)
